instructions/docs/add-the-encryption-sdk.md

```diff
@@ -229,12 +229,12 @@ Check out the code in one of the `-complete` folders to compare.
 ~/environment/workshop/exercises/java/add-esdk-complete
 ```
 
-```Javascript Node.js
+```javascript tab="JavaScript Node.JS"
 ~/environment/workshop/exercises/node-javascript/add-esdk-complete/store.js
```

Comment:
inconsistent content in tabs: actual changed files (javascript), just the directory (java, python)